internal/db/p2p_schema_root_test.go

```diff
@@ -249,9 +249,6 @@ func TestGetAllP2PCollections_WithMultipleValidCollections_ShouldSucceed(t *test
 	require.Equal(t, []string{schema2.Root, schema1.Root}, cols)
 }
 
-// This test documents that we don't allow adding p2p collections that have a policy
-// until the following is implemented:
```

Comment:
todo: Still makes sense to have the comment that this test documents that we don't allow adding p2p collections that have a policy for local acp.

Reply:
I will delete the test

- [ ] Delete unit test